packages/muelu/src/Misc/MueLu_SegregatedAFactory_decl.hpp

```diff
@@ -105,8 +105,12 @@ namespace MueLu {
 
   private:
 
-    //! Generating factory of input variable
-    mutable RCP<const FactoryBase> mapFact_;
+    void BuildBasedOnBlockmap(Level& currentLevel) const;
+
+    void BuildBasedOnMapPair(Level& currentLevel) const;
+
+    RCP<const Map> CreateRedundantMaps(
+            Teuchos::RCP<const Teuchos::Comm<int>> comm, Teuchos::RCP<const Map> localDropMap) const;
```

Comment:
Instead of passing the `comm` through the function interface, you could probably just extract it from the `localDropMap` inside the function.

Reply:
Done.